components/binary_sensor/dfrobot_sen0395.rst

```diff
@@ -0,0 +1,274 @@
+DFRobot mmWave Radar
+====================
+
+.. seo::
+    :description: Instructions for setting up DFRobot mmWave Radar
+    :image: dfrobot_sen0395.jpg
+    :keywords: mmWave
+
+The `DFRobot mmWave Radar <https://wiki.dfrobot.com/mmWave_Radar_Human_Presence_Detection_SKU_SEN0395>`__
+(aka ``SEN0395``) is a 24GHz radar which can be used for human presence detection. It can detect tiny movements
+and compared to a PIR sensor **it can detect presence continuously**. This can be useful, for example, to turn
+the lights on when you enter a room, keep them on as long as you are there (without waving your hands at the
+sensor) and turn them off almost immediately after you leave the room.
+
+It is possible to use this sensor with only a single GPIO pin; however, if you wish to change its settings,
+a :doc:`/components/uart` component (and its requisite GPIO pins) is required in your device's configuration.
+
+.. figure:: images/dfrobot_sen0395-full.jpg
+    :align: center
+    :width: 75%
+
+    DFRobot mmWave Radar / presence detection sensor
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    uart:
+      tx_pin: GPIO13
+      rx_pin: GPIO14
+      baud_rate: 115200
```

Comment:
Its preferred to not include this as it forces the user to go to the uart docs and read those.
```suggestion
```